src/aspire/utils/misc.py

```diff
@@ -121,57 +120,65 @@ def gaussian_1d(size, mu=0, sigma=1, peak=1, dtype=np.float64):
 
     p = (g["x"] - mu) ** 2 / (2 * sigma**2)
 
-    return (peak * np.exp(-p)).astype(dtype, copy=False)
+    return np.exp(-p).astype(dtype, copy=False)
 
 
-def gaussian_2d(size, x0=0, y0=0, sigma_x=1, sigma_y=1, peak=1, dtype=np.float64):
+def gaussian_2d(size, mu=(0, 0), sigma=(1, 1), dtype=np.float64):
     """
     Returns a 2d Gaussian in a square 2d numpy array.
 
-    Default is a centered disc of spread=peak=1.
+    Default is a centered disc of spread=1.
```

Comment:
Not really a “disc”? Also, by `spread`, do we mean `sigma`? Might just be better to have the formula here (and for the others).

Reply:
Ok, reworded and added formulas. Sphinx supports LaTex math formulas. Spruced things up a bit:
![Screen Shot 2022-07-28 at 10 02 23 AM](https://user-images.githubusercontent.com/66811911/181524761-0da99249-a49f-43ab-83a4-2833e6282dfb.png)

Reply:
Looks great!